features/FEATURE_BLE/ble/CharacteristicDescriptorDiscovery.h

```diff
@@ -111,8 +111,8 @@ class CharacteristicDescriptorDiscovery {
      * GattClient::discoverCharacteristicDescriptors
      * DiscoveredCharacteristic::discoverDescriptors
      */
-    typedef FunctionPointerWithContext<const DiscoveryCallbackParams_t*>
-        DiscoveryCallback_t;
+    typedef FunctionPointerWithContext<const DiscoveryCallbackParams_t *>
+    DiscoveryCallback_t;
```

Comment:
Please keep the indentation as the expression is not complete.

Reply:
https://sourceforge.net/p/astyle/tickets/37/ - this is the bug there, will check if there is any fix